src/gui_common/CustomDropDown.cs

```diff
@@ -0,0 +1,179 @@
+using System.Collections.Generic;
+using Godot;
+
+/// <summary>
+///   Basically a class that inherits MenuButton, but with extra popup menu functionality
```

Comment:
Summary should not begin with "basically" or "a class". Should look something like "A custom drop down implemented through MenuButton, but --"